README.md

```diff
@@ -33,6 +33,17 @@ You can find the [most up to date list of all of the new packages on our page](h
 
 Last stable versions of packages that have been provided for usage with Azure and are production-ready. These libraries provide you with similar functionalities to the Preview ones as they allow you to use and consume existing resources and interact with them, for example: upload a blob. They might not implement the [guidelines](https://azure.github.io/azure-sdk/python_introduction.html) or have the same feature set as the Novemeber releases. They do however offer wider coverage of services.
 
+### Management: New Releases
```

Comment:
You'll also have to update the links in the Packages Available to point to these sections instead (L20-22)

Reply:
hmm, strange, I thought I updated these, thanks for spotting :)